index.ts

```diff
@@ -181,4 +193,7 @@ async function main() {
     console.log('System online!');
 }
 
+if (!Config.DEBUG_LOGGING) {
+    console.debug = function() {}
```

Comment:
Please don't overwrite default JS functions. I would just export a debug log function from a util file that depends on this variable set or not.